dpdispatcher/dp_cloud_server.py

```diff
@@ -31,6 +31,8 @@ def __init__(self, context):
         phone = context.remote_profile.get("phone", None)
         username = context.remote_profile.get("username", None)
         password = context.remote_profile.get("password", None)
+        self.retry_count = context.remote_profile.get("retry_count", 3)
+        self.ignore_exit_code = context.remote_profile.get("ignore_exit_code", 0)
```

Comment:
Documentation should be added if users provide these parameters.

Reply:
Done